source/history.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var noop = function () {
```

Comment:
Yes. Modules (even the small ones) hide the complexibility (which is good), and `lodash.noop` won’t add a lot of payload. I’m a fan of [Sindre Sorhus’s approach](https://github.com/sindresorhus/ama/issues/10#issuecomment-117766328) to tiny modules, and I believe the size of the problem doesn’t matter.